static/app/views/settings/projectSourceMaps/projectSourceMaps.tsx

```diff
@@ -70,7 +70,7 @@ function SourceMapsTableRow({
         {idColumnDetails}
       </IDColumn>
       <ArtifactsTotalColumn>
-        <Count value={fileCount} />
+        {!isEmptyReleaseBundle && <Count value={fileCount} />}
```

Comment:
```jsx
        {isEmptyReleaseBundle ? (
          <Tooltip title={t('No bundle connected to this release')}>
            <IconWarning color="warning" size="sm" />
          </Tooltip>
        ) : <Count value={fileCount} />}
```

And remove `IconWrapper` from the bottom of the file. It's not useful if we dont have any text on the side.

Reply:
Great! Thanks